react/week2/src/components/Counter.js

```diff
@@ -5,10 +5,13 @@ function Counter() {
   const [count, setCount] = useState(0);
 
   useEffect(() => {
-    setTimeout(() => {
-      setCount(prev => prev + 1);
-    }, 1000);
-  });
+    const timer = setTimeout(() => {
+            setCount(count => count + 1);
+        }, 1000);    
+    return (() => {

```

Comment:
I added  [count]); to the end of  useEffect and looks ok

Reply:
Yes but a better solution is to move it out as we talked about in class. In programming, the optimal approach is always to isolate the logic so that changes in one place does not affect the rest of the code

Reply:
Otherwise its looking very good. 👍

Reply:
thanks, I will change it and push to master!